bubblewrap.c

```diff
@@ -1473,6 +1492,13 @@ main (int    argc,
 
       /* Initial launched process, wait for exec:ed command to exit */
 
+      if (opt_block_fd != -1)
+        {
+          char b[1];
+          read (opt_block_fd, b, 1);
+          close (opt_block_fd);
+        }
+
       /* We don't need any caps in the launcher, drop them immediately. */
```

Comment:
Of course, the grand-child still has caps at this point, so we'll still block that one.
Do you need to block the operation this early (i.e. before the sandbox setup). Maybe we can do the blocking on block_fd after the child is set up fully and has dropped caps? I.e. say just before we loaded the seccomp data?